### DISCUSSION TITLE
Per project notes?

### DISCUSSION THREAD

**Discussion by Ciel-MC** [nvim-neorg/neorg#1283] [Q&A]
I just learned about mind.nvim(well, it’s continuation) from TWiN, and I find it’s ability to have a .mind folder in a project and essentially have a project-local note tree very useful, but after a bit of searching, I don’t think Neorg has support for something like that? Am I just missing something or is this indeed not a feature yet?

**Comment by pysan3** [comment]
`:Neorg index` will open index.norg at the cwd.

**Reply by Ciel-MC** [reply]
That would be something like project_name/index.neorg right? I don’t want it to be in the way of the project, like have it be in a dot folder

**Reply by pysan3** [reply]
I globally gitignore /index.norg and that should align well with other modules as well. If you want something different, change `dirman.config.index` or you'd need to implement your own external module. 

One other option would be to define a relative path as one of the workspace, which will work if you open a neovim instance for each directory and do not change cwd afterwards. 

```lua
dirman.config.workspaces = {
  --- others
  current = vim.fn.fnamemodify("./.norg/", ":p"),
}
```

Which will be calculated when neovim is loaded (or when lazy loaded) and will be used throughout that instance.
